debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/elements/adapters/VariableColumnPresentation.java

```diff
@@ -23,35 +23,6 @@
  */
 public class VariableColumnPresentation extends AbstractColumnPresentation {
 
-	/**
-	 * Constant identifier for the default variable column presentation.
-	 * @deprecated Replaced by {@link IDebugUIConstants#COLUMN_PRESENTATION_ID_VARIABLE}
-	 */
-	@Deprecated
-	public final static String DEFAULT_VARIABLE_COLUMN_PRESENTATION = IDebugUIConstants.COLUMN_PRESENTATION_ID_VARIABLE;
-
-	/**
-	 * Default column identifiers
-	 * @deprecated Replaced by {@link IDebugUIConstants#COLUMN_ID_VARIABLE_NAME}
-	 */
-	@Deprecated
-	public final static String COLUMN_VARIABLE_NAME = IDebugUIConstants.COLUMN_ID_VARIABLE_NAME;
```

Comment:
This would break CDT, see 
https://github.com/eclipse-cdt/cdt/blob/fd56a4ee112974c345bbb67fa939dea178494c00/dsf/org.eclipse.cdt.dsf.ui/src/org/eclipse/cdt/dsf/debug/ui/viewmodel/IDebugVMConstants.java#L43

@jonahgraham : FYI.

I believe we first should make sure CDT would remove the reference to this constant. Already shipped CDT versions should be fine, as the constant value should be inlined by compiler.